src/codeu/chat/server/Server.java

```diff
@@ -70,9 +80,15 @@ public Server(final Uuid id, final Secret secret, final Relay relay) {
 
     this.id = id;
     this.secret = secret;
-    this.controller = new Controller(id, model);
+
+    BlockingQueue<Writeable> blockq = new LinkedBlockingQueue<Writeable>();
```

Comment:
nit: there doesn't seem to be a reason to instantiate the queue out here and pass it to the file writer. Prefer to hide implementation details like this by moving queue instantiation into FileWriter.